net-misc/ipsvd/ipsvd-1.0.0-r3.ebuild

```
@@ -0,0 +1,51 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs flag-o-matic
+
+DESCRIPTION="ipsvd is a set of internet protocol service daemons for Unix"
+HOMEPAGE="http://smarden.org/ipsvd/"
+SRC_URI="http://smarden.org/ipsvd/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static"
+
+DEPEND=""
+RDEPEND=""
+
+S="${WORKDIR}/net/${P}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-fix-parallel-make.diff
+	"${FILESDIR}"/${PN}-1.0.0-fix-musl-clang-16.patch
+)
+
+src_configure() {
+	cd "${S}"/src
+	if use static ; then
+		append-ldflags -static
+	fi
+
+	echo "$(tc-getCC) ${CFLAGS}"  > conf-cc
+	echo "$(tc-getCC) ${LDFLAGS}" > conf-ld
+}
+
+src_compile() {
+	cd "${S}"/src
```

```suggestion
	cd "${S}"/src || die
```
For error-handling.

Then again I wonder if it's better if you just modify `S` to include `/src`? Although in `src_install` there's some `do*` calls for `../src/` so... :\